Library/Homebrew/shims/scm/git

```diff
@@ -92,7 +91,7 @@ case "$(lowercase "$SCM_FILE")" in
     ;;
 esac
 
-brew_version="$(quiet_safe_cd "$SCM_DIR/../../../../bin" && pwd -P)/$SCM_FILE"
+brew_version="$(quiet_safe_cd "$HOMEBREW_PREFIX/bin" && pwd -P)/$SCM_FILE"
```

Comment:
It probably wouldn't, right. Even if there's an assumed fallback path that works for the default install and a custom location install that would be sufficient.